packages/codegen/src/EntityDbMetadata.ts

```diff
@@ -455,9 +455,9 @@ function newEnumField(config: Config, entity: Entity, r: M2ORelation, enums: Enu
   const columnType = (column.type.shortName || column.type.name) as DatabaseColumnType;
   const fieldName = enumFieldName(column.name);
   const enumName = tableToEntityName(config, r.targetTable);
-  const enumType = imp(`${enumName}@./entities`);
-  const enumDetailType = imp(`${plural(enumName)}@./entities`);
-  const enumDetailsType = imp(`${enumName}Details@./entities`);
+  const enumType = imp(`${enumName}@./entities.ts`);
```

Comment:
I was wondering "why are all of the existing test projects not having `.ts` added to their output?" and realized/forgot that ts-poet has the `importExtensions=false` to strip extensions--that was handy! :-)